### PR TITLE
curator library membership ingestion

### PR DESCRIPTION
**Pull request by stkem**
@yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/common/conf/evolutions/shoebox/246.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+CREATE TABLE curator_library_membership_info (
+    id bigint(20) NOT NULL AUTO_INCREMENT,
+    created_at datetime NOT NULL,
+    updated_at datetime NOT NULL,
+    state varchar(20) NOT NULL,
+
+    user_id bigint(20) NOT NULL,
+    library_id bigint(20) NOT NULL,
+    library_access VARCHAR(64) NOT NULL,
```
Comment (yingjieMiao) [inline kifi-backend/modules/common/conf/evolutions/shoebox/246.sql]: this `varchar(20)` on shoebox

**Comment by yingjieMiao** [comment]
do you have a test for ingestion ?  lgtm otherwise.

File: kifi-backend/modules/common/conf/evolutions/shoebox/246.sql (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/common/conf/evolutions/shoebox/246.sql]: oh, good find. fixing.

**Comment by stkem** [comment]
test is in the works. Will add one before merging.

**Pull request merged**